moduleroot/CONTRIBUTING.md.erb

```diff
@@ -135,12 +135,22 @@ Our puppet modules provide [`Gemfile`](./Gemfile)s which can tell a ruby
 package manager such as [bundler](https://bundler.io/) what Ruby packages,
 or Gems, are required to build, develop, and test this software.
 
-Please make sure you have [bundler installed](https://bundler.io/#getting-started)
-on your system, then use it to install all dependencies needed for this project,
-by running
+**Prerequisites**
+1. Make sure you have [bundler installed](https://bundler.io/#getting-started)
+on your system. If you are using Fedora, you can get `bundler` using
+   ```shell
+     sudo dnf install rubygem-bundler
```

Comment:
You can keep the indenting at the same level as the backticks. This is true for a few cases in this file, but to keep it brief I won't repeat the comment every time.